<launch>
 <arg name="rvizconfig" default="$(find bombel)/rviz/dkin.rviz" />	
  <param name="robot_description" command="$(find xacro)/xacro.py $(find bombel)/urdf/simple.urdf" />
  <param name="use_gui" value="true"/>
  
  <param name="d1" value="0.2005" />
  <param name="a2" value="0.1575" />
  <param name="a3" value="0.18334" />
  <param name="a4" value="0.14616" />
  <param name="theta4" value="0.78539816339" />

  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />

  <!--<node name="DKIN" pkg="bombel" type="dkin.py" output="screen"/>	-->
  
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />


</launch>
